wolfcrypt/test/test.c

```diff
@@ -10268,6 +10268,9 @@ int rsa_no_pad_test(void)
     ret = 0;
 
 exit_rsa_nopadding:
+    XFREE(tmp, HEAP_HINT, DYNAMIC_TYPE_TMP_BUFFER);
```

Comment:
Can you also fix the `return -6900;` case above at line 10114?

Reply:
Absolutely, great catch!